gradle/versions.gradle

```diff
@@ -8,7 +8,7 @@ ext {
     buildToolsVersion = "28.0.3"
 
     // Plugins
-    gradleVersion = '3.2.1'
+    gradleVersion = '3.3.1'
```

Comment:
Was going to do this. Nice catch. Where can I add thumbs up emoji.

Reply:
Right here! 💯